docs/source/federation/implementing.md

```diff
@@ -2,23 +2,24 @@
 title: Implementing a federated graph
 description: Putting the pieces together
 ---
-Apollo Federation is made up of two parts:
-* Federated services, which are standalone parts of the graph
-* A gateway which composes the overall schema and executes federated queries
 
-To be part of a federated graph, a microservice implements the Apollo Federation spec which exposes its capabilities to tooling and the gateway. The federated service can extend types from other services and add its own types that can be extended.
+An Apollo Federation architecture consists of:
 
-Collectively, federated services form a composed graph. This composition is done by a gateway which knows how to take an incoming operation and turn it into a plan of fetches to downstream services. The gateway orchestrates requests, merges the data and errors together, and forms the overall result to send back to the client.
+* A collection of **implementing services** that each define a distinct GraphQL schema
+* A **gateway** that composes the distinct schemas into a **federated data graph** and executes queries across that graph
 
-Let's take a look at how to get a federated graph up and running. We'll start by preparing an existing service for federation, followed by setting up a gateway in front of it.
+To be part of a federated graph, an implementing service must conform to the Apollo Federation specification, which exposes the service's capabilities to the gateway,
+as well as to tools like Apollo Graph Manager. A service can **extend** GraphQL types that are defined by _other_ services, and it can define types for other services to extend.
```

Comment:
Might be a good moment to mention that implementing services may be written in any language